mifosng-android/src/main/java/com/mifos/mifosxdroid/online/collectionsheetindividual/IndividualCollectionSheetPresenter.java

```diff
@@ -252,10 +252,12 @@ public void call(PaymentTypeOptions paymentTypeOption) {
                 .subscribe(new Action1<ClientCollectionSheet>() {
                     @Override
                     public void call(ClientCollectionSheet client) {
```

Comment:
`ClientCollectionSheet` object refers to the `Client` specific for CollectionSheet. Since there were already some `Client` models, I named this one which is specific to the CollectionSheet, as `ClientCollectionSheet`.  That's why I named the variable client.